src/Raven.Client/Documents/Commands/Batches/BatchCommand.cs

```diff
@@ -61,6 +70,17 @@ public override HttpRequestMessage CreateRequest(ServerNode node, out string url
                 })
             };
 
+            if (_attachmentStreams != null && _attachmentStreams.Count > 0)
+            {
+                var multipartContent = new MultipartContent();
+                multipartContent.Add(request.Content);
+                foreach (var stream in _attachmentStreams)
+                {
+                    multipartContent.Add(new GzipStreamContent(stream, "AttachmentStream"));
```

Comment:
We don't want to do compress on a per part basis. In general, we'll let users handle that externally instead of forcing it in our code.
In some cases, it is cheaper to send the data uncompressed than pay the compression costs.